utils/ips.go

```diff
@@ -187,6 +204,14 @@ func parseInterfaceSpec(spec string) (interfaceSpec, error) {
 	if spec == "inet6" {
 		return inetInterfaceSpec{Spec: spec, Name: "*", IPv6: true}, nil
 	}
+	if strings.HasPrefix(spec, "static:") {
+		ip := strings.SplitAfter(spec, "static:")
+		nip := net.ParseIP(ip[1])
+		if nip == nil {
+			return nil, fmt.Errorf("Unable to parse static ip %s in %s", ip[0], spec)
```

Comment:
Agreed.  Just don't want the static IP parser to trap potentially valid interfaces.